src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const longDays = 7;
+  const shortDays = 3;
+  const longDiscount = 50;
+  const shortDiscount = 20;
+  const price = cost * days;
+
+  if (days >= longDays) {
+    return price - longDiscount;
+  } else if (days >= shortDays) {
+    return price - shortDiscount;
+  }
+
+  return price;
```

Comment:
The function and variable names are descriptive and explain their purpose well. However, constants should be used instead of literals in expressions, and the use of 'else' after 'if' with 'return' is unnecessary and should be removed to simplify the control flow.